src/app/features/tech-record/components/tyres-search/tyres-search.component.scss

```diff
@@ -0,0 +1,10 @@
+.search-btn {
+  background-color: #1d70b8;
+  color: #fff;
+  border: none;
+  padding: 0.4rem 0.7rem 0.3rem 0.5rem;
```

Comment:
is it possible to use the responsive spacing mixing from govuk library?

Reply:
Spacing for the icon wasn't the problem, the size didn't conform to it's counterpart. (suggestions on how to fix properly welcome)